src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -313,6 +313,16 @@ public static void TestSubtract_DateTime(DateTime dateTime1, DateTime dateTime2,
     }
 
     [Fact]
+    public static void TestSubtract_DateTime_Invalid()
+    {
+        DateTime date1 = DateTime.MinValue.ToLocalTime();
+        Assert.Throws<ArgumentOutOfRangeException>(() => date1.Subtract(new TimeSpan(365, 0, 0, 0)));
+
+        DateTime date2 = DateTime.MaxValue.ToLocalTime();
```

Comment:
That code being hit from this method call is dependent on the the current timezone though, so it's not the best test for it. One of those cases where code being covered isn't the whole story :)